### DISCUSSION TITLE
TinyUSB devices will not SUPPORT AS KEYBOARD USB1.1 IN ESP32-S2

### DISCUSSION THREAD

**Discussion by benabramz1** [hathach/tinyusb#1633] [Q&A]
Hello friends
I have a problem with the keyboard.
I want to connect as a keyboard to a device that only supports USB1.1.
The microcontroller I use is ESP32-S2 with tinyusb. I have changed CFG_TUD_ENDPOINT0_SIZE in tusbconfig to 8, but No connection was made.

**Comment by chegewara** [comment]
I am suspecting its the problem with speed. You device may support only low speed keyboards.

**Reply by benabramz1** [reply]
thanks a lot chegewara
Do you suggest a way to solve this problem?
I have used arduino platform for this and I want to send some special keys to the device.
